app/models/document_capture_session.rb

```diff
@@ -3,6 +3,16 @@ class DocumentCaptureSession < ApplicationRecord
 
   belongs_to :user
 
+  def self.create_by_user_id(user_id, hash = {})
+    return unless user_id
+    reuse_session = DocumentCaptureSession.
+      find_by(user_id: user_id) || DocumentCaptureSession.new(user_id: user_id)
```

Comment:
what about `first_or_create`?

```suggestion
    reuse_session = DocumentCaptureSession.where(user_id: user_id).first_or_create
```